experimenter/experimenter/nimbus_ui_new/tests/test_views.py

```diff
@@ -1288,6 +1288,80 @@ def test_post_updates_metrics_and_segments(self):
         self.assertEqual(experiment.segments, [segment1.slug, segment2.slug])
 
 
+class TestLaunchViews(AuthTestCase):
+    def setUp(self):
+        super().setUp()
+        self.experiment = NimbusExperimentFactory.create()
+
+    def test_draft_to_preview(self):
```

Comment:
Yep nice succinct little tests, but we should explicitly set the three status fields values before we call into the form, ie we're asserting on their values after we call but not before so the experiment might've already had that value and the form could not write it and the test would still pass 🙏